src/app/shared/components/dual-slider/dual-slider.component.ts

```diff
@@ -15,24 +15,53 @@ import {
 import { Options } from 'ng5-slider';
 
 
+/**
+ * Component containing a button that when clicked will show a slider popover.
+ */
 @Component({
   selector: 'ccf-dual-slider',
   templateUrl: './dual-slider.component.html',
   styleUrls: ['./dual-slider.component.scss']
 })
 export class DualSliderComponent implements OnDestroy, OnChanges {
+  /**
+   * Reference to the template for the slider popover.
+   */
   @ViewChild(CdkPortal, { static: true }) popoverPortal: CdkPortal;
+
+  /**
+   * Reference to the popover element.
+   * This is undefined until the slider popover is initialized.
+   */
   @ViewChild('popover', { read: ElementRef, static: false }) popoverElement: ElementRef;
+
   @Input() label: string;
   @Input() valueRange: number[];
   @Input() selection: number[];
   @Output() selectionChange = new EventEmitter<number[]>();
 
+
+  /**
+   * Determines whether slider popover is shown
+   */
   isSliderOpen = false;
+
+  /**
+   * Slider options.
+   */
   options: Options;
+
+  /**
+   * Values bound to the slider's low and high pointer values.
```

Comment:
JSDocs will not pick up that you are referring to two variables here. each variable needs its own doc string.